src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -359,11 +365,15 @@ ucs_status_t uct_ud_verbs_ep_put_short(uct_ep_h tl_ep,
 static UCS_F_ALWAYS_INLINE unsigned
 uct_ud_verbs_iface_poll_tx(uct_ud_verbs_iface_t *iface, int is_async)
 {
+    unsigned num_wcs =
+            uct_ib_iface_device(&iface->super.super)->ordered_send_comp ?
+                    1 :
+                    iface->super.super.config.tx_max_poll;
```

Comment:
why don't we check several wcs for ib as well?

Reply:
I am trying to preserve original behaviour, if we support ordered send completion (non-EFA case), we already occasionally request completion and implicitly assume that the previous posts are already completed.

I guess it was assumed that it was enough to poll once since anyways completion received is already a fraction of the total number of posts.